### PR TITLE
badges added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: badges/badges.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    
+                <div class="component-details">
+                    <h1 class="component-heading">Badges on Icon</h1>
+                    <p class="component-info">Badges scale to match the size of the immediate parent element by using relative font sizing.</p>
+                    <p class="component-info">Here are sothe examples of badges with/without number at top/bottom of icon!</p>
+                </div>
+                    <div class="flex-flow-row avatar-examples avatar-texts">
+                        <div class="badge-container">
+                            <i class="fa fa-bell fa-5x"></i>
+                            <span class="badge icon-badge badge-bottom">10</span>
```
Comment (Shivani-1524) [inline badges/badges.html:73]: The text isn't centered, would look a lot better if it is.

File: badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+@import "../avatar/avatar.css";
+
+*{
+    --online-color: #108a2c;
```
Comment (Shivani-1524) [inline badges/badges.css]: Just a suggestion, for the colors used on badges to be bright especially those used for status depiction.

**Comment by Sahil25061999** [comment]
looks good to me can be merged.

File: badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    height: 2rem;
+    background-color: var(--online-color);
+}
+.badge-dnd{
+    bottom: 1rem;
+    right: 6px;
+    width: 1.75rem;
+    height: 1.75rem;   
+    background-color: var(--dnd-color);   
+}
```
Comment (rahulk31) [inline badges/badges.css]: I have a suggestion here to have a single class with width, height and positioning for badges in general. And then to have separate classes with colors. Thereafter you can incorporate these classes depending on the usecases.

**Review by rahulk31 — COMMENTED** [review]
UI Suggestion: If you are planning to have a blue bg for these badges(attached), please make the text color as lighter to as something white. So that it is readable.

image

Overall looks good to me.

File: badges/badges.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+
+            <div class="avatar-container component-container">
+                <div class="component-details">
+                    <h1 class="component-heading">Badges</h1>
+                    <p class="component-info">Documentation for badges, a labeling component.Badges scale to match the size of the immediate parent element by using relative font sizing.</p>
+                </div>
+                <div class="avatar-examples avatar-images flex-flow-row">
+                    <div class="badge-container">
+                        <img class="avatar avatar-lg" src="../images/avatar.jpg">
```
Comment (mehulsatardekar) [inline badges/badges.html:44]: Just a suggestion add an **alt attribute** on images basically it helps to describe the images

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
For Status badges, you can use box-shadow property with white color on status circles for better UI.

Before box-shadow
Capture-Before



After box-shadow
Capture-After

File: badges/badges.css (hunk 3 above)
Comment (swstica) [reply badges/badges.css]: cool, will do.

File: badges/badges.html (hunk 4 above)
Comment (swstica) [reply badges/badges.html:44]: Thanks, will make the changes

**Pull request merged**